simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/handlers/ServerMessageHandler.java

```diff
@@ -4,14 +4,12 @@
 import com.evbox.everon.ocpp.simulator.station.StationMessageSender;
 import com.evbox.everon.ocpp.simulator.station.StationState;
 import com.evbox.everon.ocpp.simulator.station.exceptions.BadServerResponseException;
-import com.evbox.everon.ocpp.simulator.station.handlers.ocpp.GetVariablesRequestHandler;
-import com.evbox.everon.ocpp.simulator.station.handlers.ocpp.OcppRequestHandler;
-import com.evbox.everon.ocpp.simulator.station.handlers.ocpp.ResetRequestHandler;
-import com.evbox.everon.ocpp.simulator.station.handlers.ocpp.SetVariablesRequestHandler;
+import com.evbox.everon.ocpp.simulator.station.handlers.ocpp.*;
```

Comment:
in Everon we don't allow star imports. Shall we keep the same policy here?

Reply:
Default intellij idea formatting. I dont see the reason to forbid that.